From 947c553ff92b6edbc6f5a9f43172ebf7e58a1d21 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?P=C3=A1draig=20Brady?= <P@draigBrady.com>
Date: Fri, 26 Jun 2020 19:57:09 +0100
Subject: [PATCH 09/13] tests: fix false failure with valgrind and reflink

* tests/cp/fiemap-FMR.sh: Avoid FICLONE ioctl,
which would avoid the point of the test (fiemap testing).
Also it avoids a valgrind bug with this ioctl:
https://bugs.kde.org/show_bug.cgi?id=397605
---
 tests/cp/fiemap-FMR.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/cp/fiemap-FMR.sh b/tests/cp/fiemap-FMR.sh
index 29be5ee07..42b244f30 100755
--- a/tests/cp/fiemap-FMR.sh
+++ b/tests/cp/fiemap-FMR.sh
@@ -24,7 +24,7 @@ require_perl_
 
 $PERL -e 'for (1..600) { sysseek (*STDOUT, 4096, 1)' \
   -e '&& syswrite (*STDOUT, "a" x 1024) or die "$!"}' > j || fail=1
-valgrind --quiet --error-exitcode=3 cp j j2 || fail=1
+valgrind --quiet --error-exitcode=3 cp --reflink=never j j2 || fail=1
 cmp j j2 || fail=1
 
 Exit $fail
-- 
2.27.0

